src/components/Image/Image.st.css

```diff
@@ -12,11 +12,19 @@
 }
 
 .root {
-  -st-states: preload, loaded;
+  -st-states: resize(enum(contain, cover)), preload, loaded;
 
   display: grid;
 }
 
+.root:resize(contain) .absoluteImage, .root:resize(contain) .relativeImage, .root:resize(contain) .placeholder {
+  object-fit: contain;
```

Comment:
This isn't supported by older browsers.
I guess it's fine, as long as it doesn't break in older browsers

Reply:
Yeah, IE11 isn't supported, but it's pretty much the case with CSS variables support as well...
AFAIR the viewer will never support IE11, so I think it's okay.

Reply:
Not exactly.
Currently TPA's also run on bolt, especially in the editor where TB isn't supported yet.
That's why we don't use css variables yet as well.

In general we should support ie11, but it's also ok if ie11 will have an inferior experience, without breakage though.

Reply:
I tested using Browserling on IE9 - it seems the browser just displays it as a regular image and ignores the property:
![image](https://user-images.githubusercontent.com/7141972/103781154-9e5cac00-503e-11eb-960a-71e0d367cada.png)

In a normal browser, it should look like this:
![image](https://user-images.githubusercontent.com/7141972/103781306-d368fe80-503e-11eb-9b4d-0040afacb6d7.png)